test/gpu/ops_test.cpp

```diff
@@ -1034,26 +1034,63 @@ struct test_layernorm : verify_program<test_layernorm>
     {
         migraphx::program p;
         std::vector<size_t> dims{1, 1, 5};
-        auto x        = p.add_parameter("x", migraphx::shape{migraphx::shape::float_type, dims});
-        auto scale    = p.add_parameter("scale", migraphx::shape{migraphx::shape::float_type, {5}});
-        auto bias     = p.add_parameter("bias", migraphx::shape{migraphx::shape::float_type, {5}});
+        auto x = p.add_parameter("x", migraphx::shape{migraphx::shape::float_type, dims});
```

Comment:
The only difference between this and the other test is `dims`. Can we write a function that adds the instructions for layernorm and takes `dims` as a parameter?